doc/foreign-code.rst

```diff
@@ -74,16 +74,18 @@ build. Note that this method can be used to build other things, not
 just C libraries.
 
 To do that, follow the following procedure:
+
 - put all the foreign code in a sub-directory
 - tell Dune not to interpret configuration files in this directory via
   an :ref:`ignored_subdirs <dune-ignored_subdirs>` stanza
 - write a custom rule that:
-  + depend on this directory recursively via :ref:`source_tree`
-  + invoke the external build system
-  + copy the C archive files (``.a``, ``.so``, ...) in main library
+
+  - depend on this directory recursively via :ref:`source_tree <source_tree>`
```

Comment:
```suggestion
  - depends on this directory recursively via :ref:`source_tree <source_tree>`
```

Reply:
It might be good to make another pull request with these suggestions, since I think they're not really related to the changes I made.